index.html

```diff
@@ -165,8 +167,72 @@ <h1>Skills</h1>
                     </ul>
                 </div>
             </div>
-                
+
+        </section>
+        <section class="getting-started">
+            <div class="content">
+                <h1>Getting Started</h1>
+                <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut
+                    labore
+                    et dolore magna.</p>
+                <form action="https://formspree.io/f/123" method="post">
+                    <div class="text-input user-name">
+                        <label for="user-name">Your name</label>
+                        <input type="text" name="name" placeholder="John Doe" id="user-name" required maxlength="30" />
+                    </div>
+                    <div class="text-input user-email">
+                        <label for="user-email">Your email</label>
+                        <input type="email" name="email" placeholder="john.doe@mail.com" id="user-email" />
+                    </div>
+                    <div class="text-input user-message">
+                        <label>
+                            <textarea id="msg" name="user-message" placeholder="Enter your message here..."
+                                maxlength="500"></textarea>
+                        </label>
+                    </div>
+                    <div class="collab">
+                        <button type="button" class="btn-primary-dark">Start Collaboration</button>
```

Comment:
- [ ] All Your input fields should have the `required `attribute to activate the validation on submit. please add a `required `attribute so that you form `inputs `will be validated, or user can not send an empty fields .